AzureStore.UI/app/src/components/Basket/Basket.tsx

```diff
@@ -96,20 +106,23 @@ export default class Basket extends React.Component<Props> {
       this.props.submitOrder(order);
     }
 
-    getTotal(items: any): number {
+    getTotal(items: OrderItem[]): number {
       let total = 0;
-      items.forEach((item: any) => total+=item.product.amount*item.quantity);
+      items.forEach((item: any) => total+=item.product.price*item.quantity);
```

Comment:
idk why, but many js tutorials recommend to use `.reduce()` function instead of forEach
```suggestion
    const total = items.reduce((result, curr) => result + curr.price*cur.quantity);
```